packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -152,6 +152,30 @@ flutter:
       ProcessManager: () => FakeProcessManager.any(),
     });
 
+    testUsingContext('throws ToolExit when directory entry contains invalid characters', () async {
+      globals.fs.file('.packages').createSync();
+      globals.fs.file('pubspec.yaml')
```

Comment:
Let's not introduce any usage of globals if we can avoid it.

```suggestion
      testFileSystem.file('.packages').createSync();
      testFileSystem.file('pubspec.yaml')
```